src/db/index.js

```diff
@@ -0,0 +1,54 @@
+class DB {
+    key;
+    schema;
+
+    constructor(key, schema) {
+        if (typeof key !== 'string' || typeof schema !== 'object' || key.length === 0 || schema.length === 0) {
```

Comment:
Please check here if the `schema` is an `Array` or not. In this check, even an object (ex: `{}`) will pass as a schema. 
To check for array you can use `Array.isArray(schema)`.